.github/workflows/release.yaml

```diff
@@ -14,10 +14,12 @@ on:  # yamllint disable-line rule:truthy
           - major
           - minor
           - patch
+  repository_dispatch:
+    types: [release-go-libs]
 jobs:
   csm-release:
     uses: dell/common-github-actions/.github/workflows/csm-release-libs.yaml@main
     name: Release Go Client Libraries
     with:
-      version: ${{ github.event.inputs.option }}
+      version: "${{ github.event.inputs.version || 'minor' }}"
```

Comment:
Should be inputs.option. Version is no longer applicable

Reply:
@khareRajshree Could you check the comment from Shayna?

Reply:
Updated in recent commit.